src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesCollectionConfiguration.cs

```diff
@@ -22,20 +22,31 @@ public class TimeSeriesCollectionConfiguration : IDynamicJson
         /// </summary>
         public RawTimeSeriesPolicy RawPolicy { get; set; } = RawTimeSeriesPolicy.Default;
 
-        internal void Initialize()
+        internal void ValidateAndInitialize()
         {
             if (Policies.Count == 0)
                 return;
 
             Policies.Sort(TimeSeriesDownSamplePolicyComparer.Instance);
 
+            RawPolicy.RetentionTime.AssertMonthOrSeconds();
```

Comment:
I wonder if we should convey implementation detail on what we assert ;-) it is impl details for me

Reply:
No, this is fine